src/languageHandler.tsx

```diff
@@ -93,7 +93,7 @@ export class UserFriendlyError extends Error {
 
 export function getUserLanguage(): string {
     const language = SettingsStore.getValue("language", null, /*excludeDefault:*/ true);
-    if (language) {
+    if (typeof language === "string" && !!language) {
```

Comment:
oh man. wtf happens when you implicitly cast a string to bool? Can you either make this explicit or comment it, to save me having to look it up?